src/classes/audioContextWrapper.tsx

```diff
@@ -47,6 +52,7 @@ class AudioContextWrapper {
 
     this.masterGainNode.connect(this.audioContext.destination);
     this.waveform = getWave(waveformId);
+    this.wavetable = WavetableCache.getSingleton().get("organ") ?? Wavetable.createEmpty();
```

Comment:
This seems like a logical skew. Isn't the waveform and wavetable mutually exclusive?